2018-13-2018_VA Flagship_Mobile_Application/Deliverables/Skylight Digital LLC/mobile_application_comparative_analysis/public_sector_apps.csv

```diff
@@ -0,0 +1,9 @@
+Agency,App Name,Link,App Description,Platforms,More Links,
+IRS,IRS2Go ,https://play.google.com/store/apps/details?id=gov.irs&hl=en_US,"Check refund status, make payments, find free tax preparation assistance, sign up for helpful tax tips, generate a login security code, and follow the latest news from the IRS.","iOS, Android",,
+HUD,HUD Resource Locator,https://itunes.apple.com/us/app/hud-resource-locator/id1018440488?mt=8,"Helps users discover commonly requested federal housing resources within their community. Get information on many housing HUD programs, directly contact specific resource providers, and more.","iOS, Android",,
+USPS,USPS Mobile,https://itunes.apple.com/us/app/usps-mobile/id339597578?mt=8,"Calculate shipping prices, find a Post Office, look up a ZIP Code, schedule a next-day pickup, request USPS to hold your mail, and more. ","iOS, Android",,
+National Highway Traffic Safety Administration,Safer Ride,https://one.nhtsa.gov/link/saferride/,Keep drunk drivers off our roads by allowing users to identify their location and call a taxi or friend to pick them up. ,"iOS, Android",,
+FEMA,FEMA,https://www.fema.gov/mobile-app,"Real time alerts, tips on what to do before, during, and after for more 20 types of disasters. Locate shelters and find out where to contact FEMA at disaster recovery centers. ","iOS, Android",,
+TSA,MyTSA,https://www.tsa.gov/mobile,"Search items are allowed through a checkpoint, view delays and current weather conditions by airport, see how busy the airport is likely to be on your specific day and time of travel based on historical data, view the TSA Guide, see TSA Pre✓ locations, request live assistance from TSA through the AskTSA social media integration. ","iOS, Android",,
+State Department,Smart Traveler,https://2009-2017.state.gov/r/pa/ei/rls/dos/165020.htm,"The official State Department app for U.S. travelers invites you to see the world with easy access to frequently updated official country information, travel alerts, travel warnings, maps, U.S. embassy locations, and more. ","iOS, Android",,
+Navy,Ready2Serve,https://itunes.apple.com/us/app/ready-2-serve/id1092691249,"Provides the Navy’s Reserve Force the ability to deliver and receive digital information and access essential services (like access to orders and opportunities, email and calendar and a support center) from anywhere with a mobile CAC reader issued by the Navy Reserve to authorized users during the registration process.","iOS, Android",Navy’s promotional video,A post about the process by 18f
```

Comment:
I think "Navy’s promotional video" and "A post about the process by 18f" were supposed to be links?

Reply:
should be fixed now